File: BackendPhotoItemsApp/Models/TypePropertySet.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BackendPhotoItemsApp.Models
{
    public partial class TypePropertySet
    {
        public int Id { get; set; }
        public int MediaTypeId { get; set; }
        public int PropertyId { get; set; }

        public virtual Type MediaType { get; set; }
        public virtual Property Property { get; set; }
    }
}

File: frontend-photo-items-app/src/models/TypePropertySet.ts
import Property from "./Property";
import Type from "./Type";

export default class TypePropertySet {

  public id: number = 0;
  public mediaTypeId: number = 0;
  public propertyId: number = 0;
  public value: string = "";

  //Navigations
  public mediaTypeNavigation: Type = new Type();
  public propertyNavigation: Property = new Property();

  constructor(typePropertySet?: Partial<TypePropertySet>) {
    Object.assign(this, typePropertySet);
  }

}